updates/28/updates.sql

```diff
@@ -72,3 +72,11 @@ INSERT INTO assets(identifier, name, type) VALUES("eip155:8453/erc20:0xCb327b99f
 *
 INSERT INTO assets(identifier, name, type) VALUES("eip155:100/erc20:0x5Cb9073902F2035222B9749F8fB0c9BFe5527108", "Monerium GBPe", "C"); INSERT INTO evm_tokens(identifier, token_kind, chain, address, decimals, protocol) VALUES("eip155:100/erc20:0x5Cb9073902F2035222B9749F8fB0c9BFe5527108", "A", 100, "0x5Cb9073902F2035222B9749F8fB0c9BFe5527108", 18, ""); INSERT INTO common_asset_details(identifier, symbol, coingecko, cryptocompare, forked, started, swapped_for) VALUES("eip155:100/erc20:0x5Cb9073902F2035222B9749F8fB0c9BFe5527108", "GBPe", "monerium-gbp-emoney", "", NULL, 1669766840, NULL);
 *
+UPDATE common_asset_details SET cryptocompare="LINK" WHERE identifier="eip155:100/erc20:0xE2e73A1c69ecF83F464EFCE6A5be353a37cA09b2";
```

Comment:
The Link assets had PLINK as an asset id for some reason in cryptocompare which is wrong.

Same for some USDC assets having USDBC.